wscript

```diff
@@ -39,8 +40,8 @@ leveldb_version = '1.20'
 lmdb_version = '0.9.21'
 # Ubuntu 18.04 has libmongoc 1.9.2
 libmongoc_version = '1.9.0'
-# Ubuntu 18.04 has SQLite 3.22.0
```

Comment:
Why did you remove this?

Reply:
That was not intended, I think this happened during merge.